frontend/src/components/Mobile/BottomNavBar.tsx

```diff
@@ -85,14 +84,14 @@ function BottomNavBar() {
             }}
           ></TabBar.Item> */}
           <TabBar.Item
-            key="Supper"
-            title="Supper"
-            icon={<StyledButton src={supperIconUnselected} />}
-            selectedIcon={<StyledButton src={supperIconSelected} />}
+            key="Gym"
+            title="Gym"
+            icon={<StyledButton src={gymIcon} />}
+            selectedIcon={<StyledButton src={gymIcon} />}
             selected={activeTabIndex() === 3}
             // badge={'test'}
```

Comment:
For this and https://github.com/rhdevs/rhapp/pull/636/files#r835946588, I think we should change, but because if we do change we might want to change all the other TabBars at once, I think might be better to change it in a seperate standalone PR?